### PR TITLE
Let send a message with the failure to the user who triggered this change

### PR DESCRIPTION
**Pull request by xabolcs**
Fix for Issue #80.

### CODE REVIEW

**Comment by xabolcs** [comment]
@whimboo please review!

File: bot.js [hunk 1]
```diff
@@ -527,6 +531,13 @@ function saveData(datastore, data) {
   fs.writeFile(filename, data, function(err){
     if (err) {
       console.error("Error writing " + filename);
+      if (typeof callback == "function") {
+        try {
+          callback(err);
```
Comment (whimboo) [inline bot.js]: Hm, what is `callback`? I don't see it defined anywhere in this module.

Comment (xabolcs) [inline bot.js]: Ooops. :) Too few copy-paste.

Comment (xabolcs) [inline bot.js]: It's a function parameter. From now. ;)

Comment (whimboo) [inline bot.js]: Now I see the logic you had in mind. Do we really need this callback parameter? I would hard-code the code right here with a single error message. Or would there be a problem?

Comment (xabolcs) [inline bot.js]: `client.say(from, message);` won't work at that point. `from` would be undefined.
At least `from` should be passed. Passing an `onError` callback is more flexible, IMHO.

Comment (xabolcs) [inline bot.js]: It's OK to me to to that hard-coding, if you really want to go that way.

Comment (whimboo) [inline bot.js]: Nah, keep it as it is. Lets squash the commits and I will land it later. Thanks!

**Comment by xabolcs** [comment]
Commits squashed! Please review the commit message!

**Comment by whimboo** [comment]
Updated commit message and merged to master as https://github.com/mozilla/testdaybot/commit/a81e10d2970cac7501257b2bb92adbf1d3bb9202. Thanks!

**Pull request closed without merging**